README.md

```diff
@@ -1,19 +1,22 @@
-Nit is an expressive language with a script-like syntax, a friendly type-system and aims at elegance, simplicity and intuitiveness.
+[![Stories in Ready](https://badge.waffle.io/nitlang/nit.png?label=todo&title=TODO)](https://waffle.io/nitlang/nit)
+[![Gitter](https://badges.gitter.im/nitlang/nit.svg)](https://gitter.im/nitlang/nit?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge)
 
-Nit has a simple straightforward style and can usually be picked up quickly, particularly by anyone who has programmed before.
-While object-oriented, it allows procedural styles.
+**Nit is an expressive language with a script-like syntax, a friendly type-system and aims at elegance, simplicity and intuitiveness.**
```

Comment:
Because `<strong>` exists to apply strong emphasis to a piece of text, not to apply a particular style to a whole text. Please do not abuse of the semantic elements and respect the [separation of presentation and content](https://en.wikipedia.org/wiki/Separation_of_presentation_and_content).